types/hashmap/hashmap-tests.ts

```diff
@@ -12,7 +12,8 @@ emptyMap.set("foo", 123).set("foo2", 234);
 emptyMap.multi("foo3", 345, "foo4", 456).multi("foo5", 567, "foo6", "678");
 emptyMap.copy(filledMap).copy(copiedMap);
 
-const value: number = emptyMap.get("foo");
+const value: number = emptyMap.get("foo") as number;
+const missingValue: undefined = emptyMap.get("bar") as undefined;
```

Comment:
Shouldn't this be asserted? Something like
```typescript
import assert = require("assert");
/* ... */
assert(typeof emptyMap.get("foo") === "number");
assert(typeof emptyMap.get("bar") === "undefined");
```

<sub><sup>To be honest I do not remember if test files are executed or only checked for compile errors...</sup></sub>

Reply:
The CI system for `DefinitelyTyped` actually runs the test script so there's no need for asserting.

You can check the logs [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51282/checks?check_run_id=1918105305) (the `Run npm run test-all` step)

Reply:
Surely the CI performs `npm run test-all`, but my point is if files are _actually run_. I just checked, and in fact only `dtslint` in run, therefore is just a matter of compile-time checks.

In this case `assert`ing is obviously useless, and at the same time `as` casts would not work if the static type would be wrong (i.e: changing `get` to return `TValue | null` would raise an error because `null as undefined` is not allowed).

So, for me it's fine! :blush: